Solution/ContosoProject/Domain/Entities/Order.cs

```diff
@@ -6,7 +6,34 @@
 
 namespace Domain.Entities
 {
-    public class Order
+    public class Order : BaseEntity
     {
+        public Customer customer;
```

Comment:
I don't think that all variables must be properties, so I named them as just an ordinary variables, without capitalization